core/src/test/java/org/infinispan/globalstate/GlobalStateBackwardsCompatibilityTest.java

```diff
@@ -0,0 +1,85 @@
+package org.infinispan.globalstate;
+
+import static org.infinispan.commons.test.CommonsTestingUtil.tmpDirectory;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.IntStream;
+
+import org.infinispan.commons.util.Util;
+import org.infinispan.configuration.cache.CacheMode;
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.configuration.global.GlobalConfigurationBuilder;
+import org.infinispan.manager.EmbeddedCacheManager;
+import org.infinispan.test.MultipleCacheManagersTest;
+import org.infinispan.topology.PersistentUUID;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.Test;
+
+/**
+ * ISPN-12667 A test to ensure that global state properties from previous Infinispan majors are compatible.
+ *
+ * @author Ryan Emerson
+ * @since 12.0
+ */
+public class GlobalStateBackwardsCompatibilityTest extends MultipleCacheManagersTest {
```

Comment:
The test isn't running.
```suggestion
@Test(groups = "functional", testName = "globalstate.GlobalStateBackwardsCompatibilityTest")
public class GlobalStateBackwardsCompatibilityTest extends MultipleCacheManagersTest {
```

Reply:
I swear I forgot the `@Test` annotation every time! Or copy and paste and forget to update the `testName` :face_with_head_bandage: